docs/source/getting-started.rst

```diff
@@ -231,8 +231,8 @@ Item validation
 
 Item validators allows you to match your returned items with predetermined structure
 ensuring that all fields contains data in the expected format. Spidermon allows
-you to choose between schematics_ or `JSON Schema`_ to define the structure
-of your item.
+you to choose from schematics_,`JSON Schema`_ or `cerberus`_ to define structure and
```

Comment:
What about “[…] to define the validation rules of your items.”?

Reply:
@Gallaecio I am not sure I get what you meant.

Reply:
I meant, what about changing “to define structure and validation tool needed for your item.” to “to define the validation rules of your items.”. It’s mostly an :lipstick: aesthetic change, though.

Reply:
Good change, I see what you mean. Implemented.